src/routes/admins.js

```diff
@@ -38,8 +33,8 @@ export default (apiServer, maxFileSize) => {
     return res
   }
 
-  apiServer.get('/v1/admins/', async req => {
-    allowAccessTo(req, secrets, [{ type: 'admin' }])
+  apiServer.get('/v1/system-admins/', async req => {
+    allowAccessTo(req, process.env.SECRETS.split(' '), [{ type: 'admin' }])
```

Comment:
Please, do it at the top of the file, after the imports. This way, it's doing an array operation on every http request.